test/dedup-id/.gitignore

```diff
@@ -0,0 +1,4 @@
+lib/**
```

Comment:
Is there really a `lib` folder in the module. I believe this to be a copy-paste from another module. There used to be a `lib` folder when `babel` was in place, to my knowledge (I was not yet involved).